js/ct-checkusers.js

```diff
@@ -48,9 +48,18 @@ window.apbct = window.apbct || {};
     usersChecker.setListeners = () => {
         $('#check_spam_users').click(() => {
             usersChecker.improvedCheck = $("#ct_impspamcheck_checkbox").is(":checked");
-            usersChecker.clearUserCheckerResults().then(() => {
+
+            usersChecker.clearUserCheckerResults()
+                .then((response) => {
+                    // Clear Frontend Results
+                    const responseObject = JSON.parse(response);
```

Comment:
`JSON.parse()` has to be wrapped by `try...catch`

Reply:
DONE